app/services/foreman_ansible/ansible_info.rb

```diff
@@ -1,7 +1,11 @@
 module ForemanAnsible
   class AnsibleInfo < ::HostInfo::Provider
     def host_info
-      { 'parameters' => ansible_params }
+      if Setting[:ansible_parameterized_roles_in_enc]
+        { 'roles' => ansible_roles }
```

Comment:
I would change the key to `'ansible_roles'` to make it more obvious.

Reply:
Makes sense, I'll update it.